### ISSUE TITLE
Create application docs with moin, publish by loading to readthedocs

### ISSUE THREAD

**Issue by RogerHaase** [moinwiki/moin#1656]
A possible approach:

* create application docs using any of moin's markup languages
* use dump-html cli to convert to html files
*  use https://stackoverflow.com/questions/64518816/readthedocs-publishing-pre-built-html-pages-to-readthedocs to load to readthedocs

**Comment by ThomasWaldmann** [comment]
If the goal is publishing docs to readthedocs, using sphinx might be the simpler and well-established approach.

**Comment by renato67** [comment]
yes (and no :-) ). I would undestand if this could be a correct tool-chain to create documentation:
- Moinmoin2 (to collect doc from various people)
- export it to Sphinx
- import on ReadtheDoc
- export in HTML or PDF

but for what I've unserstand I can export from moinmoin and import it directly to Readthedocs. avoiding Sphinx. Is it?

Renato

**Comment by renato67** [comment]
ok,
so Moin2 has the task of collecting the doc that various people do (with his powerful to handle users)
and Readthedoc can transform this documentation to various format.
Is it so?

**Comment by renato67** [comment]
I would like to know if this is the way the 2 SW collaborate or am I wrong

**Comment by sfermigier** [comment]
> I would like to know if this is the way the 2 SW collaborate or am I wrong

This is A way the two CAN (or COULD) collaborate. Obviously one can be used without the other one.

**Comment by renato67** [comment]
of course...but coul be that the scenario? (moin2 collect doc and RTD publish it in various format?

**Comment by renato67** [comment]
still not clear...
Do RTD and Sphinx make the same work? If yes, I can realize the entire doc (made by various author) with a wiki (moin2 or something else) then paste it to RTD and obtain various format.
Isn't it?

**Comment by ReimarBauer** [comment]
When you make the installation and develop doc and usage doc with the wiki, why do you want to use RTD?

Usually I ask developers on a PR to update also the docs in the repo. Writing docs is usually part of the development process. When you move the whole doc maintaining process to the wiki, then it is usually not the developer who changes it.

**Comment by renato67** [comment]
@ReimanBauer. So you are telling that I can build docs using a repo (i.e GIT) and directly RTD, without a wiki. But i.e. if I would start a chain where everybody (developer and normal employ) could write doc not just for program documentation,  I could manage the "fishing" of docs by a wiki and paste the result to RTD that can produce more kind of docs.
Right?

**Comment by gmilde** [comment]
RTD uses Sphinx to create the HTML pages from *reStructuredText* or *Markdown* sources fetched from a Git repository.
So you can keep the documentation sources alongside the code in your repo and
* use Sphinx locally to generate the documentation to ship with releases
* use RTD to publish "living documentation" from the sources.

>if I would start a chain where everybody (developer and normal employ) could write doc not just for program documentation, I could manage the "fishing" of docs by a wiki 

Moin2 sources may be written in (or converted to) reStructuredText or Markdown, so you can easily download the sources in one of these formats and incorporate them into the documentation sources in your repo.

Caveats:
* Moin format conversion is lossy, so for "documentation fishing" it's best to use rST or markdown markup in the wiki.
* Sphinx defines many extensions to both, reStructuredText and Markdown that are not supported in Moin.
